### PR TITLE
Add Andy Matuschak article to the catalog

### PR DESCRIPTION
**Pull request by akkartik**

### CODE REVIEW

File: data/catalog.yaml [hunk 1]
```diff
@@ -735,3 +735,38 @@ works:
       "Related to situated software ([<a
       href='https://www.gwern.net/docs/technology/2004-03-30-shirky-situatedsoftware.html'>1</a>][<a
       href='https://buttondown.email/hillelwayne/archive/situated-software'>2</a>])"
+armories:
```
Comment (jryans) [inline data/catalog.yaml:738]: I noticed this entry didn't show up in the preview, and I believe it's this missing indentation. I'm starting to regret using a YAML here... 😓 

```suggestion
  armories:
```

**Review by jryans — CHANGES_REQUESTED** [review]
Thanks for curating this, looks great overall! 😄

Just a small syntax tweak needed, and then it's ready to go.

File: data/catalog.yaml (hunk 1 above)
Comment (akkartik) [reply data/catalog.yaml]: Thank you! I was wondering what was wrong..

**Review by jryans — APPROVED** [review]
Looks great, thanks again! 😄

**Pull request merged**